skrub/datasets/_fetching.py

```diff
@@ -190,15 +190,6 @@ def _fetch_openml_dataset(
     if not details_gz_path.is_file() or not features_gz_path.is_file():
         # If the details file or the features file don't exist,
         # download the dataset.
-        warnings.warn(
```

Comment:
I believe scikit-learn's openml fetcher makes the download in a temporary directory, then renames it if the checksum is correct to avoid corrupted downloads, so this warning may be superfluous.

there is a similar warning in another function that does not follow the same strategy, I'll open another issue about fixing the fetcher code before removing that second warning